lib/hobo-serenity.rb

```diff
@@ -36,7 +36,7 @@ def create_document(document_path, template_name, delete_odt=false, create_pdf=t
       
     # Convert to PDF via LibreOffice 
     if create_pdf == true
-      odt_to_pdf_cmd = "soffice --headless --convert-to pdf --outdir '#{[Rails.root, document_folder].join('/')}' '#{[Rails.root, odt_path].join('/')}' &>>#{[Rails.root,'log',Rails.env + ".log"].join('/')}"
+      odt_to_pdf_cmd = "soffice --headless --convert-to pdf --outdir '#{[Rails.root, document_folder].join('/')}' '#{[Rails.root, odt_path].join('/')}' >>#{[Rails.root,'log',Rails.env + ".log"].join('/')}"
```

Comment:
I've re-checked that this little ampersand can cause catastrophic failure :(
So this fix is right :)